### PR TITLE
Using GlobalTag not JECfromFile

### PR DESCRIPTION
**Pull request by DouglasBurns**
Use global tags for JEC and not .DB files

### CODE REVIEW

File: python/Jets_Setup_cff.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def setup_jets(process, cms, options, postfix="PFlow"):
     if usePrivateSQlite:
         from CondCore.DBCommon.CondDBSetup_cfi import *
         import os
-        era = "Summer15_50nsV4_"
+        era = "Summer15_50nsV4_" #now version 5
```
Comment (kreczko) [inline python/Jets_Setup_cff.py]: why not change it to 5 then?

File: python/MET_Setup_cff.py [hunk 2]
```diff
@@ -100,6 +93,16 @@ def setup_MET(process, cms, options, postfix="PFlow"):
     # process.shiftedPatJetEnUpNoHF.jetCorrLabelUpToL3Res = cms.InputTag("ak4PFCHSL1FastL2L3Corrector")
 
 
+    # recalculate MET without HF
```
Comment (kreczko) [inline python/MET_Setup_cff.py]: is this not included in MiniAODv2?

Comment (DouglasBurns) [inline python/MET_Setup_cff.py]: yes but at the time I tried to make sure it was backwards compatible so older files could run over it - i can remove

File: python/Jets_Setup_cff.py (hunk 1 above)
Comment (DouglasBurns) [inline python/Jets_Setup_cff.py]: will do - but it is of little importance anyway for the moment as these are for the JEC files.

File: python/MET_Setup_cff.py [hunk 3]
```diff
@@ -12,38 +12,32 @@ def setup_MET(process, cms, options, postfix="PFlow"):
...
-                ),
-            cms.PSet(
-                record = cms.string("JetCorrectionsRecord"),
-                tag = cms.string("JetCorrectorParametersCollection_"+era+"_AK4PFchs"),
-                label= cms.untracked.string("AK4PFchs")
-                ),
-            )
-                               )
-    process.es_prefer_jec = cms.ESPrefer("PoolDBESSource",'jec')
+    # from CondCore.DBCommon.CondDBSetup_cfi import *
```
Comment (kreczko) [inline python/MET_Setup_cff.py]: you can also just remove it instead of commenting out

File: python/Jets_Setup_cff.py [hunk 4]
```diff
@@ -38,7 +38,7 @@ def setup_jets(process, cms, options, postfix="PFlow"):
     if usePrivateSQlite:
         from CondCore.DBCommon.CondDBSetup_cfi import *
         import os
-        era = "Summer15_50nsV4_"
+        era = "Summer15_50nsV5_"
```
Comment (kreczko) [inline python/Jets_Setup_cff.py]: Should this not be ```Summer15_25nsV5```?
https://hypernews.cern.ch/HyperNews/CMS/get/jes/558.html

File: python/MET_Setup_cff.py (hunk 3 above)
Comment (DouglasBurns) [inline python/MET_Setup_cff.py]: by the looks of it we dont need to use the MET_setup_cff at all

**Comment by DouglasBurns** [comment]
I believe ready for merging

**Comment by DouglasBurns** [comment]
can be merged

**Pull request merged**